src/lib/FieldType/Mapper/ImageFormMapper.php

```diff
@@ -68,6 +89,29 @@ public function configureOptions(OptionsResolver $resolver)
                 'translation_domain' => 'ibexa_content_type',
             ]);
     }
+
+    /**
+     * @return array<string>
+     */
+    private function getMimeTypesChoiceList(): array
+    {
+        $mimeTypeChoiceList = [];
+        foreach ($this->allowedMimeTypes as $mimeType) {
+            $extensions = implode(
+                ', ',
+                array_map(
+                    static function (string $extension): string {
+                        return '*.' . $extension;
+                    },
+                    $this->mimeTypes->getExtensions($mimeType)
+                )
+            );
+            $label = explode('image/', $mimeType);
```

Comment:
Are we sure that explode always returns at least two elements within an array?

Reply:
There should be always two elements because mime type have format `image/<type>`